export const handleError = (message, e) => {
  alert(message);
  console.log(e);
};